quiche/src/recovery/gcongestion/recovery.rs

```diff
@@ -477,7 +479,7 @@ impl GRecovery {
         let mut duration = self.pto() * (1 << self.pto_count);
 
         // Arm PTO from now when there are no inflight packets.
-        if self.bytes_in_flight == 0 {
+        if self.bytes_in_flight.get() == 0 {
```

Comment:
super nit: maybe a `is_zero()` would be useful here

Reply:
Done.